lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_method.rb

```diff
@@ -140,7 +140,7 @@ def self.with_automation_env
       Bundler.with_clean_env do
         begin
           backup = ENV.to_hash
-          gem_paths = (backup["GEM_PATH"].split(File::PATH_SEPARATOR) << bundle_path).uniq
+          gem_paths = (Gem.path << bundle_path).uniq
```

Comment:
@bdunne The `<<` operator is going to modify the `Gem.path` array, is that an intended action here?  Should this be something like `(Gem.path + [bundle_path])` instead?